css/style.css

```diff
@@ -149,3 +100,35 @@ header {
     justify-content: space-between;
 }
 
+/* media - queries */
+
+/* scroll bar */
```

Comment:
Seems these comments have not been removed or moved else where, take care of it

Reply:
Resolved.